skippy-core/src/main/java/io/skippy/core/HashUtil.java

```
@@ -88,14 +73,11 @@ private static String fullHash(byte[] data) {
         }
     }
 
-    private static byte[] getBytecodeWithoutDebugInformation(Path classFile) {
+    private static byte[] readByteCode(Path classFile) {
         try (var inputStream = newInputStream(classFile)) {
-            var classWriter = new ClassWriter(Opcodes.ASM9);
-            var classVisitor = new ClassVisitor(Opcodes.ASM9, classWriter) {};
-            new ClassReader(inputStream).accept(classVisitor, ClassReader.SKIP_DEBUG);
-            return classWriter.toByteArray();
+            return inputStream.readAllBytes();
```

What about `Files.readAllBytes(Path)`? Both solutions would fail in case of reading a class that is way too large (> 2GiB), but `Files.readAllBytes(Path)` would fail early.